tekton-catalog/pipeline-loops/pkg/reconciler/pipelinelooprun/pipelinelooprun.go

```diff
@@ -78,6 +78,10 @@ const (
 
 	// LabelKeyWorkflowRunId is the label identifier a pipelinerun is managed by the Kubeflow Pipeline persistent agent.
 	LabelKeyWorkflowRunId = "pipeline/runid"
+
+	DefaultNestedStackDepth = 100
```

Comment:
Can we set the `DefaultNestedStackDepth` to 30? I do realize when we have more than 30 stack, the k8s api-server start getting very slow.

Reply:
Yes, it was my guess. Thanks for clarifying the number.